package-lock.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
we need to delete this from this PR, any npm deps need to be installed upstream in the emergence-skeleton repo where `package.json` and the rest of the base cypress environment live

in fact... everything in this PR should be going there instead, please close this PR and reopen one there. Changes up `package.json` and `package-lock.json` should all be applied through running `npm` commands in that repo.

After that PR merges I'll handle running the reformat up the whole stack as I bump versions